src/rpc/register.h

```diff
@@ -10,23 +10,23 @@
 class CRPCTable;
 
 /** Register block chain RPC commands */
-void RegisterBlockchainRPCCommands(CRPCTable &tableRPC);
+void RegisterBlockchainRPCCommands(CRPCTable &t);
```

Comment:
I don't think consistency is useful here; using full names is nicer for documentation purposes.